pom.xml

```diff
@@ -598,6 +610,10 @@
                     <groupId>msv</groupId>
                     <artifactId>xsdlib</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>xerces</groupId>
+                    <artifactId>xercesImpl</artifactId>
+                </exclusion>
```

Comment:
this exclusion isn't truly necessary, but we are explicitly pulling in xerces (line 562), so it felt right to exclude it here.